.github/workflows/ci.yml

```diff
@@ -20,18 +20,47 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.8"
           - "3.9"
           - "3.10"
+          - "3.11"
+          - "3.12"
+          - "3.13"
         django-version:
-          - django~=3.2.0
-          - django~=4.1.0
           - django~=4.2.0
-
+          - django~=5.0.0
+          - django~=5.1.0
+          - django~=5.2rc0
+        exclude:
+          # Django 5.0+ requires Python >=3.10
+          - python-version: "3.9"
+            django-version: django~=5.0.0
+          - python-version: "3.9"
+            django-version: django~=5.1.0
+          - python-version: "3.9"
+            django-version: django~=5.2rc0
+          # Python 3.13 supported only in Django >=5.1.3
+          - python-version: "3.13"
+            django-version: django~=4.2.0
+          - python-version: "3.13"
+            django-version: django~=5.0.0
+    
     steps:
     - uses: actions/checkout@v4
       with:
         ref: ${{ github.event.pull_request.head.sha }}
+    
+    - name: Cache APT packages
+      uses: actions/cache@v4
+      with:
+        path: /var/cache/apt/archives
+        key: apt-${{ runner.os }}-${{ hashFiles('.github/workflows/ci.yml') }}
+        restore-keys: |
+          apt-${{ runner.os }}-
+    
+    - name: Disable man page auto-update
+      run: |
+        echo 'set man-db/auto-update false' | sudo debconf-communicate >/dev/null
+        sudo dpkg-reconfigure man-db
```

Comment:
Why do we need this change?

Reply:
@codesankalp we're adding this to all the repos to speed up builds